app/controllers/teams_controller.rb

```diff
@@ -37,9 +37,28 @@ def destroy
     end
   end
 
+  def members
+    if @team
+      members = @team.users
+      render json: members
+    else
+      head  status: :not_found
+    end
+  end
+
+  def vacations
+    if @team
+      vacations = @team.users.collect do |user|
+        user.vacation_requests.requested_accepted_inprogress
```

Comment:
There would be also a lot of sql queries:
to fix it you can select all `requested_accepted_inprogress` by special team, to find by team you need to join `VacationRequest` table with `Users` and `TeamRoles` tables